### ISSUE TITLE
AutoFocus prop not working with field

### ISSUE THREAD

**Issue by bhd-vasyl** [erikras/redux-form#3554]
Custom Field 
```
return (
    <div className={className}>
      <div className={touched && (error &&
        styles.errorInput
      )}>
      <label className={styles.labelText}>{label}</label>
      <Input
        {...input}
        theme={isTheme}
        widthPercent={widthPercent}
        type={type}
        value={value || input.value}
        placeholder={placeholder}
        autoFocus={autoFocus}
        inputRef={fieldRef}
        />
        {touched && !textNotNeeds && (error &&
        <span className={styles.errorText}>{error}</span>)}
      </div>
    </div>
  );
```

and it's field 
 ```
<Field
     theme="boldText"
     type="text"
     name="name"
     textNotNeeds={true}
     component={_Field}
     validate={[required, isLongerThan128]}
     onChange={title.onChange}
     fieldRef={fieldRef}
     autoFocus={true}
     placeholder="New campaign" />
```

But it's seems like no focus attribute settled up. Is there any other ways to set `autoFocus`  property?

**Comment by kreba** [comment]
I had a similar issue but managed to solve it. My mistake was that I did not pass through `autoFocus` explicitly. You seem to do it properly but I can't tell for sure since you don't show the initialization of the local variables that you use for rendering. It might be worth noting that `autoFocus` is included in the props that `_Field` receives but not in the `input` prop.

I did something along the lines of
```
const { input, meta, ...passthrough } = props;
return <_Field {...passthrough} {...input} type="text" />;
```

HTH
